features/docs/getting_started.feature

```diff
@@ -9,7 +9,7 @@ Feature: Getting started
     When I run `cucumber`
     Then it should fail with:
       """
-      No such file or directory - features. You can use `cucumber --init` to get started.
+      Feature folder not found at
       """
```

Comment:
@nodo I like kind of 2. "kind of 2" means when I removed "File.expand_path", I did not need to add `tmp/features' to features/docs/getting_started.feature. Check my additional commit, I am going to push it right now.

I prefer `File.expand_path`.
But I like you would add `File.expand_path` by yourself later, if you need it.
Because right now I want to pass all the tests on Ruby 2.4.0.